plugin/pkg/admission/podtolerationrestriction/admission.go

```diff
@@ -162,6 +164,21 @@ func (p *podTolerationsPlugin) Admit(a admission.Attributes) error {
 		}
 	}
 
+	if qoshelper.GetPodQOS(pod) != api.PodQOSBestEffort {
```

Comment:
its a doc issue.  you basically will want this controller to be the one that appears right before `ResourceQuota`